config/prow/autobump-config/prow-component-autobump-config.yaml

```diff
@@ -23,6 +23,17 @@ prefixes:
     repo: "https://github.com/kubernetes/test-infra"
```

Comment:
BTW: this file still says `repo: "https://github.com/kubernetes/test-infra"` but I guess it doesn't matter 😄

Reply:
That's wrong in the upstream config too 😅 
Now it's fixed.